.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```

Comment:
Maybe add a default label bug?

Reply:
Default label for a feature request should probably not be bug but enhancement instead right?

Reply:
yes, sorry, I copy-pasted it from the bug template